source/core/NumericString.cpp

```diff
@@ -1868,7 +1829,31 @@ VIREO_FUNCTION_SIGNATUREV(StringFormat, StringFormatStruct)
         format.AliasAssign(tempformat.Begin(), tempformat.End());
     }
     StringRef buffer = _Param(StringOut);
-    Format(&format, count, arguments, buffer);
+    Format(&format, count, arguments, buffer, null);
+    return _NextInstruction();
+}
+struct StringFormatWithErrStruct : public VarArgInstruction
+{
+    _ParamDef(StringRef, StringOut);
+    _ParamDef(StringRef, StringFormat);
+    _ParamDef(ErrorCluster, ErrClust);
+    _ParamImmediateDef(StaticTypeAndData, argument1[1]);
+    NEXT_INSTRUCTION_METHODV()
+};
+
+VIREO_FUNCTION_SIGNATUREV(StringFormatWithErr, StringFormatWithErrStruct) {
```

Comment:
I am confused. Why are we adding a new Vireo instruction with ErrorStruct but why not modify the existing one? When the next version of Vireo is used to build LabVIEW, the DFIR should be modified to generate correctly. I think that's how we have been doing and I would rather not do work twice (second time to just delete functions). "npm link" makes it easier to build LV against local builds of Vireo. So, @cglzaguilar  can just develop DFIR side of this in parallel and get the DFIR part working and give even feedback.